pkg/typeutil/convension.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package typeutil
+
+import (
+	"encoding/binary"
+
+	"github.com/pkg/errors"
+)
+
+// BytesToUint64 is transfer bytes to uint64.
```

Comment:
Seems strange. May need some help from i18n :)

Reply:
BytesToUint64 parses a slice of byte to uint64?